src/static/main.css

```diff
@@ -163,6 +167,10 @@ body {
 	display: inline-block;
 }
 
+.onclick-menu:hover{
+	cursor: pointer;
+}
+
```

Comment:
Remove this rule, please. `pointer` is the style of mouse cursor that is shown over links (👆). Our menu items (and buttons) are not links, they initiate actions, not navigation. Also (in my OS at least) the default (arrow) cursor is shown in all standard system-generated menus.